### DISCUSSION TITLE
How to add a local package to vcpkg

### DISCUSSION THREAD

**Discussion by Sallee1** [microsoft/vcpkg#33836] [Q&A]
I need to install a package through vcpkg, but there is a dependency that always fails to compile. Can I manually compile this dependency and add it to the vcpkg environment, so that it will be automatically skipped the next time I need to install this package?

**Comment by Osyotr** [comment]
It is possible. Create an overlay port which just copies your manually built files to appropriate directories.
However, if the package is in main vcpkg registry, then you can submit an issue and get some help fixing the build.
